internal/characters/yaoyao/yuegui.go

```diff
@@ -98,6 +98,7 @@ func (c *char) makeHealCB(area combat.AttackPattern, hi player.HealInfo) func(co
 			return
 		}
 		if c.Core.Combat.Player().IsWithinArea(area) {
+			hi.Target = c.Core.Player.Active()
```

Comment:
Now this will also overwrite the hi of the burst radish which is supposed to heal everyone.